generators/uscore-r4/generator.rb

```diff
@@ -31,8 +31,22 @@ def generate_search_validators(metadata)
 end
 
 def generate_tests(metadata)
+  # first isolate the profiles that don't have patient searches
+  delayed_sequences = metadata[:sequences].select do |sequence|
```

Comment:
This seems like it could be a separate method.

Reply:
Moved to another function.